com.chartboost.helium/Android~/helium-android-bridge/src/main/java/com/chartboost/heliumsdk/unity/HeliumEventProcessor.kt

```diff
@@ -0,0 +1,79 @@
+package com.chartboost.heliumsdk.unity
+
+import android.util.Log
+import com.chartboost.heliumsdk.ad.HeliumAdError
+import org.json.JSONException
+import org.json.JSONObject
+
+object HeliumEventProcessor {
+    private const val TAG = "HeliumEventProcessor"
+
+    @JvmStatic
+    fun serializeHeliumEvent(
+        placementName: String,
+        heliumAdError: HeliumAdError?,
+        eventConsumer: HeliumEventConsumer<String, Int, String>
+    ) {
+        eventConsumer.accept(placementName, heliumAdError?.getCode() ?: -1, heliumAdError?.getMessage() ?: "")
+    }
+
+    @JvmStatic
+    fun serializeHeliumBidEvent(
+        placementName: String,
+        dataMap: HashMap<String, String>,
+        eventConsumer: HeliumBidEventConsumer<String, String, String, Double>
+    ) {
+        val partnerId = dataMap["partner_id"] ?: ""
+        val auctionId = dataMap["auction-id"] ?: ""
```

Comment:
One has an underscore and one has a dash. Is that right? If so, we might want to enforce consistency so things don't break later on.

Reply:
This is from the server, I have no control over it LOL

Reply:
But yeah, I let @pleasesavemycat know about this a while ago.

Reply:
:happysad